packages/engine-client/src/types/serverExecuteTypes.ts

```diff
@@ -58,6 +59,7 @@ type EngineServerExecuteResultRequestType = {
 }[keyof EngineServerExecuteRequestByType];
 
 export interface EngineServerPlaceOrderParams {
+  id: number | null;
```

Comment:
@yruej301 what's the data type on the BE? if u64 and above, it should be a string to be serialized / deserialized properly

Reply:
it's `id: Option<u64>`

Reply:
Okay yeah these will need to be strings

Reply:
any way we can handle this on the sdk or do we need to change the backend? a simple response today

```
Place order custom id result
{
  "status": "success",
  "signature": "0x82029583b0b896b6d1f3d50559e100d7a5fb6b9a3354b128d22617d47eef5e24093837477c9cb350108fa3f39106b2b1da4924e2a7d8bbc46f239e4c268946be1b",
  "id": 100,
  "data": {
    "digest": "0xaa9e80d64b7db12a2085a926f46fbee7601a32e6c97b5dcd3894c5cf373f96ea"
  },
  "request_type": "execute_place_order",
  "orderParams": {
    "subaccountName": "default",
    "expiration": "13835058056988725377",
    "price": 3000,
    "amount": "-3500000000000000000",
    "subaccountOwner": "0x1042f83075161394Cd5bE4Ef9602b81Ed9E0563E",
    "nonce": "1789459636064092732"
  }
}
```

Reply:
or if `id` was not provided
```
{
  "status": "success",
  "signature": "0x82029583b0b896b6d1f3d50559e100d7a5fb6b9a3354b128d22617d47eef5e24093837477c9cb350108fa3f39106b2b1da4924e2a7d8bbc46f239e4c268946be1b",
  "data": {
    "digest": "0xaa9e80d64b7db12a2085a926f46fbee7601a32e6c97b5dcd3894c5cf373f96ea"
  },
  "request_type": "execute_place_order",
  "orderParams": {
    "subaccountName": "default",
    "expiration": "13835058056988725377",
    "price": 3000,
    "amount": "-3500000000000000000",
    "subaccountOwner": "0x1042f83075161394Cd5bE4Ef9602b81Ed9E0563E",
    "nonce": "1789459636064092732"
  }
}
```

Reply:
Backend would need to update - if the number is greater than https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER then `JSON.parse` will result in an incorrect n
umber

Reply:
ok sounds good, i will put out a PR for this change on the backend then. only thing is this is probably gonna be breaking for MMs who are already relying on this field so it will require comms, etc